service/client/client.js

```diff
@@ -110,8 +110,8 @@ export default class Client {
         return `${this.url}${this.urlVersion}/users`;
     }
 
-    getFilesRoute(teamId) {
-        return `${this.url}${this.urlVersion}/teams/${teamId}/files`;
+    getFilesRoute() {
```

Comment:
Could we keep that one and possibly add a ```getTeamFilesRoute```? Feel like that makes more sense because of the ```teamId``` arg.